subprojects/docs/src/docs/userguide/ant.adoc

```diff
@@ -27,173 +27,271 @@ You can use Gradle's Ant integration as a path for migrating your build from Ant
 
 [[sec:ant_properties]]
-=== Ant properties and references
+== Ant properties and references
 
 There are several ways to set an Ant property, so that the property can be used by Ant tasks. You can set the property directly on the `AntBuilder` instance. The Ant properties are also available as a Map which you can change. You can also use the Ant `property` task. Below are some examples of how to do this.
 
-++++
-<sample id="antProperties" dir="userguide/ant/properties" title="Setting an Ant property">
-            <sourcefile file="build.gradle" snippet="set-property"/>
-            <sourcefile file="build.xml" snippet="set-property"/>
-        </sample>
-++++
+=== Example: Setting an Ant property
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/ant/properties/build.gradle[tag=set-property]
+----
+
+[source,xml]
+.build.xml
+----
+include::{samplesPath}/userguide/ant/properties/build.xml[tag=setproperty]
+----
 
 Many Ant tasks set properties when they execute. There are several ways to get the value of these properties. You can get the property directly from the `AntBuilder` instance. The Ant properties are also available as a Map. Below are some examples.
 
-++++
-<sample id="antProperties" dir="userguide/ant/properties" title="Getting an Ant property">
-            <sourcefile file="build.xml" snippet="get-property"/>
-            <sourcefile file="build.gradle" snippet="get-property"/>
-        </sample>
-++++
+=== Example: Getting an Ant property
+
+[source,xml]
+.build.xml
+----
+include::{samplesPath}/userguide/ant/properties/build.xml[tag=getproperty]
+----
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/ant/properties/build.gradle[tag=get-property]
+----
 
 There are several ways to set an Ant reference:
 
-++++
-<sample id="antProperties" dir="userguide/ant/properties" title="Setting an Ant reference">
-            <sourcefile file="build.gradle" snippet="set-reference"/>
-            <sourcefile file="build.xml" snippet="set-reference"/>
-        </sample>
-++++
+=== Example: Setting an Ant reference
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/ant/properties/build.gradle[tag=set-reference]
+----
+
+[source,xml]
+.build.xml
+----
+include::{samplesPath}/userguide/ant/properties/build.xml[tag=setreference]
+----
 
 There are several ways to get an Ant reference:
 
-++++
-<sample id="antProperties" dir="userguide/ant/properties" title="Getting an Ant reference">
-            <sourcefile file="build.xml" snippet="get-reference"/>
-            <sourcefile file="build.gradle" snippet="get-reference"/>
-        </sample>
-++++
+=== Example: Getting an Ant reference
+
+[source.multi-language-sample,groovy]
+.build.gradle
```

Comment:
the order of build.gradle vs build.xml seems to have changed here. Looking at the document in its previous form, I don't think it was ever consistent (for instance, "setting an ant reference" has gradle first, while "getting an ant property" above has xml first). Maybe we should standardize it one way or the other with these changes.

Reply:
I suppose I'll just switch it back for now. Thanks